Resources/Maps/aspid.yml

```diff
@@ -49155,6 +49132,31 @@ entities:
         - 0
         - 0
         - 0
+- proto: ClosetSteelBase
+  entities:
+  - uid: 9867
+    components:
+    - type: Transform
+      pos: -34.5,-21.5
+      parent: 1
+    - type: EntityStorage
+      air:
+        volume: 200
+        immutable: False
+        temperature: 293.1496
+        moles:
+        - 3.4430928
+        - 12.952587
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0
+        - 0

```

Comment:
What's goin on here

Reply:
i dunno thats just mapping man a bunch of stuff changes lol